src/index.js

```diff
@@ -46,6 +47,7 @@ if (!isProduction) {
 }
 
 
+app.use('/api/v1', router);
```

Comment:
I suggest you use `app.use('/', router);` instead of `app.use('/api/v1', router);` and in the `src/routes/api/index.js` file you use `router.use('/api/v1', userRoutes);` This way, we are able to connect all routes in the entry file and filter routes in the `src/routes/api/index.js` file.

Reply:
ok, would update that.